tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -66,6 +95,78 @@ void test_connectivity() {
   }
 }
 
+void test_bbh_time_dependent_factory() {
+  const auto binary_compact_object =
+      TestHelpers::test_factory_creation<DomainCreator<3>>(
+          "  BinaryCompactObject:\n"
+          "    InnerRadiusObjectA: 0.2\n"
+          "    OuterRadiusObjectA: 1.0\n"
+          "    XCoordObjectA: -2.0\n"
+          "    ExciseInteriorA: true\n"
+          "    InnerRadiusObjectB: 1.0\n"
+          "    OuterRadiusObjectB: 2.0\n"
+          "    XCoordObjectB: 3.0\n"
+          "    ExciseInteriorB: true\n"
+          "    RadiusOuterCube: 22.0\n"
+          "    RadiusOuterSphere: 25.0\n"
+          "    InitialRefinement: 2\n"
+          "    InitialGridPoints: 6\n"
+          "    UseEquiangularMap: true\n"
+          "    TimeDependence:\n"
+          "      UniformTranslation:\n"
+          "        InitialTime: 1.0\n"
+          "        Velocity: [2.3, -0.3, 0.5]\n"
+          "        FunctionOfTimeNames: [TranslationX, TranslationY, "
+          "TranslationZ]");
+  const std::array<double, 4> times_to_check{{0.0, 4.4, 7.8}};
+
+  constexpr double initial_time = 0.0;
```

Comment:
Because I've forgotten: What does making this `constexpr` gain you as opposed to using `const`?

Reply:
It makes it compile-time, so it could speed up runtime (in practice in a test not really relevant :) )